src/Our.Umbraco.NestedContent/Web/UI/App_Plugins/NestedContent/Views/nestedcontent.html

```diff
@@ -25,7 +25,7 @@
 
                 </div>
 
-                <div class="nested-content__content" ng-if="$parent.realCurrentNode.id == node.id && !$parent.sorting">
+                <div class="nested-content__content" ng-show="$parent.realCurrentNode.id == node.id && !$parent.sorting">
```

Comment:
We'd already switched from using `ng-show` to `ng-if`, there was a performance impact with `ng-show` when dealing with larger lists.  See comments in PR #40 and commit 61804487b2c27dbf6d7863d79f1540da14d1ce11